.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,10 @@
 	"name": "Avalanche-DevContainer",
 
 	"build": {
-		"dockerfile": "Dockerfile"
+		"dockerfile": "Dockerfile",
+		"args": {
+			"AVALANCHEGO_VERSION": "v1.10.5"
```

Comment:
Is there anyway we can derive this from https://github.com/ava-labs/precompile-evm/blob/main/scripts/versions.sh?. I.e by running the script when building the dockerfile?

Reply:
@ceyonur We're working on this